packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -354,6 +354,10 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     );
   }
 
+  get showCopyToClipboard() {
+    return !!navigator?.clipboard?.writeText;
+  }
+
```

Comment:
It's just a question of preference, but I would see this condition and the logic in the HTML better inside the `QuanticGeneratedAnswerCopyToClipboard` component. The main point behind this is keeping the QuanticGeneratedAnswer component as simple as possible.

Reply:
My rule of thumb is to have simple child components that always render themselves, and delegate the resposibility of showing/hiding those simple components to a container component. When the visibility logic becomes too much for the top container, then I usually add an intermediate container component to deal with that part of the logic.

I don't mind moving the condition into the child component if you prefer that.